src/components/LoadingContainer/LoadingContainer.tsx

```diff
@@ -0,0 +1,31 @@
+import {type ReactNode} from 'react';
+import {cn} from '../../utils/cn';
+import {Loader} from '../Loader';
+
+import './LoadingContainer.scss';
+
+const block = cn('ydb-loading-container');
+
+interface LoadingContainerProps {
+    loader?: ReactNode;
+    children?: ReactNode;
+    loading?: boolean;
+    className?: string;
+}
+
+// Use visibility:hidden to preserve loading content size
+// Also it helps to properly render charts and virtualized elements
+// that didn't render properly with display:none since they need parent container sizes
+export const LoadingContainer = ({
```

Comment:
I'm not sure that this additional wrapper above `Loader` is really essential. I would better stick to basic library component without wrappers.